src/soc/intel/alderlake/romstage/fsp_params.c

```diff
@@ -180,6 +180,18 @@ static void fill_fspm_cpu_params(FSP_M_CONFIG *m_cfg,
 	m_cfg->PrmrrSize = get_valid_prmrr_size();
 	m_cfg->EnableC6Dram = config->enable_c6dram;
 	m_cfg->HyperThreading = get_uint_option("hyper_threading", CONFIG(FSP_HYPERTHREADING));
+
+	if (CONFIG(DRIVERS_EFI_VARIABLE_STORE) && CONFIG(FSP_HYPERTHREADING))
```

Comment:
Maybe `get_uint_option("hyper_threading", CONFIG(FSP_HYPERTHREADING))` should simply be used as a default value in `get_hyper_threading_option()`?  Should make it easier to reuse for other SOCs.

Reply:
I have hardcoded the default to true in VFR, so I will have to update DMP side if we want to depend on CONFIG(FSP_HYPERTHREADING)

Reply:
Updated DMP wit ha new PCD for HT default state. ALso moved the get_uint_option call to vc/dasharo